File: PHP PowerPoint Examples/LayoutSlidesApiDemo/getLayoutSlidesList.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\LayoutSlidesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new LayoutSlidesApi($configuration);

$name = "GetLayoutSlidesList.pptx";
$password = null;
$folder = "input";
$storage = null;
$result = $apiInstance->getLayoutSlidesList($name, $password, $folder, $storage);
print_r($result);

File: Python PowerPoint Examples/LayoutSlideApiDemo/getLayoutSlidesList.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.layout_slides_api.LayoutSlidesApi(configuration)

name = "GetLayoutSlide.pptx"
password = ""
storage = ""
folder = "input"

result = api.get_layout_slides_list(name=name, password=password, storage=storage,
                                    folder=folder)
